components/card/stories/template.js

```diff
@@ -96,6 +96,7 @@ export const Template = ({
               class=${classMap({ [`${rootClass}-coverPhoto`]: true })}
               style=${styleMap({ "background-image": `url(${image})` })}
             ></div>
+            <hr class="spectrum-Divider spectrum-Divider--sizeS spectrum-Card-divider">
```

Comment:
We should import the `Divider()` story template and use it here. Otherwise the divider CSS is not included on the page. Inspecting this `<hr>` you can see that there are no `.spectrum-Divider` styles applied. 

For reference, see where we are using `import { Template as Divider } from "@spectrum-css/divider/stories/template.js";` in some other components.

Reply:
Got it! Opened https://github.com/adobe/spectrum-css/pull/3098 to address this. Thank you!